package.json

```diff
@@ -7,7 +7,8 @@
   "main": "dist/lib/index.js",
   "types": "dist/lib/index.d.ts",
   "dependencies": {
-    "@transmute/did-key-secp256k1": "^0.2.1-unstable.35",
+    "@transmute/did-key-secp256k1": "0.2.1-unstable.35",
```

Comment:
beware that these versions are getting kinda old... also, i think technically only the key pair class is required, not the did key wrapper around it... possible can just use  https://github.com/transmute-industries/verifiable-data/blob/main/packages/json-web-signature/src/JsonWebKey.ts

Reply:
Used `@transmute/ed25519-key-pair` and `@transmute/secp256k1-key-pair` in `IonKey` class instead of their did-key counterpart: https://github.com/decentralized-identity/ion-sdk/pull/20/commits/ac8c23664031007274c68508f848a1be785b8724

However there was one other instance of `@transmute/did-key-secp256k1` that is left to substitute: https://github.com/decentralized-identity/ion-sdk/blob/main/lib/LocalSigner.ts#L23

The `ES256K` class is not present anymore in the newest versions of the secp256k1 key libraries I could find. @OR13 Do you have a suggestion for where I can find a newer version of this class for signing?

Reply:
yes :) 

https://github.com/transmute-industries/verifiable-data/blob/main/packages/jose-ld/src/JWS/jose.test.ts

https://github.com/transmute-industries/verifiable-data/blob/main/packages/jose-ld/src/JWS/__fixtures__/transmute-helper.ts

^ the reason those JOSE modules were removed is that they all did the same thing, so they were refactored to look like this... and ensure that the same code is used for each JWS, regardless of the raw signature underneath.

Reply:
Thanks! These examples were very helpful. Done: https://github.com/decentralized-identity/ion-sdk/pull/20/commits/661da321cf76dc38f835493e536adda24661cfa1